packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -401,7 +402,18 @@ const nextServerlessLoader: loader.Loader = function () {
         // make sure to set renderOpts to the correct params e.g. _params
         // if provided from worker or params if we're parsing them here
         renderOpts.params = _params || params
+        renderOpts.getFontDefinition = (fontURL) => {
+          console.log('will dist dirrrr', "${distDir}", "${canonicalBase}", "${basePath}");
+          const manifest = requireFontManifest("${distDir}", true)
```

Comment:
problem is that `distDir` comes `private-dot-next` not sure how to require the manifest when in here.

Reply:
This comes down to basically an error `private-dot-next/serverless/font-manifest.json` NOT FOUND.
what is `private-dot-next`?